tests/e2e/fixtures/plugins/faz-e2e-pmp-mock/faz-e2e-pmp-mock.php

```
@@ -0,0 +1,59 @@
+			$requested = array( absint( $levels ) );
+		}
+
+		$raw  = (string) get_option( 'faz_e2e_pmp_mock_levels', '' );
+		$mine = array_values( array_filter( array_map( 'absint', array_map( 'trim', explode( ',', $raw ) ) ) ) );
+
+		if ( empty( $mine ) ) {
+			return false;
+		}
+
+		if ( empty( $requested ) ) {
+			return ! empty( $mine );
```

PHPStan (2.1.46)

[error] Variable $mine in empty() always exists and is not falsy.


(empty.variable)